eng/targetingpacks.targets

```diff
@@ -87,6 +87,16 @@
     </PropertyGroup>
   </Target>
 
+  <!-- SDK tries to download runtime packs when RuntimeIdentifier is set, remove them from PackageDownload item. -->
+  <Target Name="RemoveRuntimePackFromDownloadItem"
+          Condition="'$(_UseLocalTargetingRuntimePack)' == 'true' and '$(RuntimeIdentifier)' != ''"
+          AfterTargets="ProcessFrameworkReferences">
+    <ItemGroup>
+      <PackageDownload Remove="@(PackageDownload)"
+                       Condition="$([System.String]::Copy('%(Identity)').StartsWith('Microsoft.NETCore.App.Runtime'))" />
+    </ItemGroup>
```

Comment:
```suggestion
      <PackageReference Remove="@(PackageDownload)"
                        Condition="'$(UsePackageDownload)' != 'true' and $([System.String]::Copy('%(Identity)').StartsWith('$(SharedFrameworkName).Runtime'))" />
    </ItemGroup>
```

Reply:
The runtime pack is not added as a `PackageReference` instead it is added as a `PackageDownload`, so leaving as is.

Reply:
It is on desktop msbuild (which means also inside VS) where PackageDownload isn't supported. This is mostly a correctness suggestion.

Reply:
Ah... I thought you were suggesting change `PackageDownload` to `PackageReference`, you're suggesting to add another remove. That makes sense. The diff here made it difficult to see.

Reply:
No worries. I made the diff as is so that you can just click on "Commit suggestion" :P

Reply:
Done.